src/main/java/com/hyperwallet/clientsdk/model/HyperwalletBankCard.java

```diff
@@ -15,14 +15,14 @@
 public class HyperwalletBankCard extends HyperwalletBaseMonitor {
 
     public enum Brand {VISA, MASTERCARD}
-
     public enum CardType {DEBIT}
-
-    private HyperwalletTransferMethod.Type type;
+    public static enum Type {BANK_CARD}
```

Comment:
Can we have String data type instead of enum here?

Reply:
Generic comment for all the enums with a single value.

Reply:
This is the extension of the existing logic from Enum. It looks like they want to avoid any other values than what are documented. Having only one value in the enum field is to ensure that same data type is used across all the methods - create, update or list. In order to make this better we can do away with the Type field totally as the users are anyway using a specific business object. But that will call for some redesign of objects. I am making this change as part of the changes in V3 for business continuity. The users are supposed to migrate to V4 soon which will avoid these hacks/ changes. Let me know if you still have any more concern.